charts/osm/templates/_helpers.tpl

```diff
@@ -8,3 +8,10 @@
 {{- $address := printf "jaeger.%s.svc.cluster.local" (include "osm.namespace" .) -}}
 {{ default $address .Values.OpenServiceMesh.tracing.address}} 
 {{- end -}}
+
+{{/* Labels to be added to all resources */}}
+{{- define "osm.labels" -}}
+app.kubernetes.io/name: openservicemesh
```

Comment:
Should this be `openservicemesh.io`?

Reply:
I'm not sure there's any specific convention here that disagrees with it as is, but I can see openservicemesh.io being a bit more clear. I've updated it.